### DISCUSSION TITLE
Welcome to ProtoTracer Discussions!

### DISCUSSION THREAD

**Discussion by coelacant1** [coelacant1/ProtoTracer#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aaronze** [comment]
👋 

I'm Awuff, Protogen Wolf from Australia. I'm a programmer.

I haven't made many contributions to the community yet, but now that I have a protogen mask I will be making contributions to the Protogen OS. I specialise in performance and optimisations, and I welcome challenges if there's an area or slow code that's causing some grief. I haven't developed on C++ and arduinos much, but I'll quickly pick it up. I appreciate any improvements or alternative suggestions as I want the best code to succeed.
